scipy/linalg/_basic.py

```diff
@@ -164,6 +167,10 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
 
     # Backwards compatibility - old keyword.
     if sym_pos:
+        message = ("The `sym_pos` keyword is deprecated and should be "
```

Comment:
In terminal messages we don't need backticks, they are too thin and looks awkward on some fonts. Use single or escape double-ticks to emphasize the parameter; `'sym_pos'`  or `\"sym_pos\"` would be sufficient.